### ISSUE TITLE
Add `vertex_submit_ext` which takes a known amount of Vertexes

### ISSUE THREAD

**Issue by sanbox-irl** [YoYoGames/GameMaker-Feature-Requests#1535]
### Is your feature request related to a problem?

Having a vertex buffer which contains more bytes than vertexes you want to actually render is now trivially possible since `vertex_update_buffer_from_buffer`. This is good and useful, but `vertex_submit` will still try to draw everything.

### Describe the solution you'd like

I'd love a `vertex_submit_ext(vtx_buf, pr_trianglelist, -1, NUMBER_OF_VERTEXES_TO_DRAW)`. The only difference is that it takes a NUMBER_OF_VERTEXES_TO_DRAW.

**Comment by stuckie** [comment]
An offset would be useful as well

**Comment by sanbox-irl** [comment]
@stuckie yeah sounds fine, since `vertex_submit` should just boil down to:

https://learn.microsoft.com/en-us/windows/win32/api/d3d11/nf-d3d11-id3d11devicecontext-draw?redirectedfrom=MSDN

and (possibly -- I can't use renderdoc on MacOS, so i'm not totally sure of GMS2's OGL impl):

https://registry.khronos.org/OpenGL-Refpages/gl4/html/glDrawArrays.xhtml

**Comment by sanbox-irl** [comment]
Certainly on dx11, `vertex_submit` is just `Draw`, which takes an offset and len
